pkg/controller/uniteddeployment/uniteddeployment_update.go

```diff
@@ -79,6 +80,17 @@ func (r *ReconcileUnitedDeployment) manageSubsets(ud *appsv1alpha1.UnitedDeploym
 	if updateErr == nil {
 		SetUnitedDeploymentCondition(newStatus, NewUnitedDeploymentCondition(appsv1alpha1.SubsetUpdated, corev1.ConditionTrue, "", ""))
 	} else {
+		// If using an Adaptive scheduling strategy, when the subset is scaled out leading to the creation of new Pods,
+		// future potential scheduling failures need to be checked for rescheduling.
+		var newPodCreated = false
```

Comment:
is it really necessary to trigger the requeue here given that manageUnschedulableStatusForExistingSubset had already push some duration time.

Reply:
1. if a duration time is pushed in manageUnschedulableStatusForExistingSubset, this will overwrite it
2. manageUnschedulableStatusForExistingSubset only pushes some duration only when an unschedulable subset is dectected or not recovered, which means for normal subsets, where scaling up happens most often, no duration will be pushed.